src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -68,6 +68,17 @@ A proof-of-stake system is kept secure by the fact that you'd need 51% of the to
 
 More on [proof-of-stake (PoS)](/developers/docs/consensus-mechanisms/pos/)
 
+### Sybil Resistance & Chain Selection
+
+Now technically, both Proof of Work (PoW) and Proof of Stake (PoS) are not consensus protocols by themselves, but they are often referred to as such for simplicity. PoW and PoS are actually Sybil Resistance mechanisms and block author selectors, they are a way to decide who is the author of the latest block. It's this mechanism combined with at least a Chain selection rule that makes up a true consensus mechanism.
```

Comment:
> Proof of Work (PoW) and Proof of Stake (PoS)

Been trying to be consistent throughout the site using `proof-of-work (PoW)` or `proof-of-stake (PoS)` (without the hyphens if at the beginning or end of a sentence). 

> Resistance

> Chain 

Should these be capitalized?

Reply:
updating, no they should not be